moduleframework/tests/static/dockerfile_lint.py

```diff
@@ -57,34 +57,75 @@ def _test_for_env_and_label(self, docker_env, docker_label, env=True):
             label_found = self.dp.get_specific_label(docker_label)
         return label_found
 
-    def _get_msg(self, label):
-        return label + " is missing in Dockerfile."
+    def _get_msg(self, label, reference=None):
+        msg = label + " is missing in Dockerfile."
+        if reference:
+            msg = msg + '\n\tMore info: ' +reference
+        return msg
 
     def test_architecture_label_exists(self):
-        self.assertTrue(self.dp.get_specific_label("architecture"),msg=self._get_msg("architecture label"))
+        """
+        :avocado: tags=rhel,fedora
+        """
+        self.assertTrue(self.dp.get_specific_label("architecture"),
+                        msg=self._get_msg("architecture label",
+                                          reference='https://docs.docker.com/engine/reference/builder/#from'))
 
     def test_name_in_env_and_label_exists(self):
+        """
+        :avocado: tags=fedora
+        """
         self.assertTrue(self.dp.get_docker_specific_env("NAME="), msg=self._get_msg("Environment variable NAME"))
-        self.assertTrue(self.dp.get_specific_label("name"), msg=self._get_msg("Label name"))
+        self.assertTrue(self.dp.get_specific_label("name"),
+                        msg=self._get_msg("Label name",
+                                          reference='https://fedoraproject.org/wiki/Container:Guidelines#LABELS'))
 
     def test_maintainer_label_exists(self):
-        self.assertTrue(self.dp.get_specific_label("maintainer"), msg=self._get_msg("Label maintainer"))
+        """
+        :avocado: tags=fedora
+        """
+        self.assertTrue(self.dp.get_specific_label("maintainer"),
+                        msg=self._get_msg("Label maintainer",
+                                          reference='https://fedoraproject.org/wiki/Container:Guidelines#LABELS'))
 
     def test_release_label_exists(self):
-        self.assertTrue(self.dp.get_specific_label("release"), msg=self._get_msg("Label release"))
+        """
+        :avocado: tags=rhel,fedora
```

Comment:
rhel is not needed. It is automatically generated by OSBS. I would prefer to remove it completely.